src/services/assets/AssetsService.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { sanitizeNumbers } from '../../sanitize/sanitizeNumbers';
+import { blockHash789629, mockApi } from '../test-helpers/mock';
+import { AssetsService } from './AssetsService';
+
+const assetsService = new AssetsService(mockApi);
+
+describe('AssetsService', () => {
+	describe('AssetsService.fetchAssetById', () => {
+		it('Should return the correct response for an AssetId', async () => {
+			const expectedResponse = {
+				at: {
+					hash:
```

Comment:
That formatting looks wonky, is this done auto?

Reply:
No its not auto-generated. It's similar structure I wrote out for `ParasService` tests [here](https://github.com/paritytech/substrate-api-sidecar/blob/master/src/services/paras/ParasService.spec.ts).

Curious how do you think i could make the structure less wonky?

Reply:
Put the value on the same line as the key? :)

Reply:
Ahh, i see what you are saying, yea definitely a little wonky. That's just the linter in this case :).